exporter/awsxrayexporter/translator/aws.go

```diff
@@ -163,6 +196,14 @@ func makeAws(attributes map[string]string, resource pdata.Resource) (map[string]
 		}
 	}
 
+	// Since we must couple log group ARNs and Log Group Names in the same CWLogs object, we first try to derive the
+	// names from the ARN, then fall back to just recording the names
+	if logGroupArns != (pdata.AnyValueArray{}) && logGroupArns.Len() > 0 {
```

Comment:
Do we need this `!=` even though we have a len check?

Reply:
Yeah, I initially didn't have it and it turns out the `Len()` method relies on an underlying pointer `orig` in the `AnyValueArray` struct, so if we don't initialize `logGroupArns` we get a nil pointer exception. I could check for nilness of `orig`, but it seemed like an implementation detail that I shouldn't be referencing directly.